spec/MongoTransform.spec.js

```diff
@@ -320,6 +320,9 @@ describe('parseObjectToMongoObjectForCreate', () => {
     done();
   });
 
+});
+
+describe_only_db('mongo')('parseObjectToMongoObjectForCreate', () => {
```

Comment:
I was thinking about an edge to edge test case, with proper data (create multiple objects and run the query). Do you think that's possible?